docker-images/jmxtrans/docker-entrypoint.sh

```diff
@@ -17,8 +16,10 @@ MONITOR_OPTS="-Dcom.sun.management.jmxremote -Dcom.sun.management.jmxremote.ssl=
               -Djava.rmi.server.hostname=${PROXY_HOST}"
 
 if [ "$1" = 'start-without-jmx' ]; then
+    # shellcheck disable=SC2086
     set /usr/bin/tini -w -e 143 -- java -server $JAVA_OPTS $JMXTRANS_OPTS $GC_OPTS $EXEC
```

Comment:
I think you can remove the sc disable line if you write:
```
set /usr/bin/tini -w -e 143 -- "java -server ${JAVA_OPTS} ${JMXTRANS_OPTS} ${GC_OPTS} ${EXEC}"
```
Might be wrong on this.

Reply:
I gave this a try and it doesn't work. Tini takes it and tries to execute it as single executable - and says that such file/directory does not exist.